.github/workflows/priority_bumper.yml

```diff
@@ -0,0 +1,34 @@
+name: Projects priority automation
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  bump_to_p2:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.comments > 10 }}
```

Comment:
Is there a way to check for the number of distinct customers instead of number of comments?

Reply:
Not directly from any of the webhook event payload. It would require a script to fetch all the comments and logins to find out distinct customers. This is one of the reason why I have used higher comment count to bump the priority.